Gemfile.lock

```diff
@@ -259,8 +259,6 @@ DEPENDENCIES
   github-pages
   html-proofer
   jekyll-archives
-  jekyll-paginate
-  jekyll-sitemap
 
```

Comment:
Did you mean to remove both of those Gems?

Reply:
Hmm no, I think it happened when running locally. Let me revert that change :+1: